.env

```diff
@@ -12,6 +12,9 @@ ENGINE_MOUNT=authorities
 FCREPO_HOST=fcrepo
 FCREPO_PORT=8080
 FCREPO_URL=http://fcrepo:8080/rest
+GOOD_JOB_CLEANUP_DISCARDED_JOBS=false
```

Comment:
If these values need to be on the server, you also should add them to the [server_name].tmpl.yml files

Reply:
@ShanaLMoore is correct here. I've added them so this can get merged and be part of the QA run